### PR TITLE
Nexus Crawl Quest

### PR DESCRIPTION
**Pull request by sLackeR-hub**
**Added**

**Creatures-**
Hea Apostate Shock Trooper
Apostate Servant
Apostate Nexus Hollow Minion
Apostate Nexus Master

**Items-**
Nexus Crawler's Mask
Apostate Master's Broken Mask
Aetheric Resonator
Contract for Nexus Crawl

**NPCs-**
Ned the Clever

**Landblocks-**
7F02 - (Apostate Nexus)

### CODE REVIEW

File: Database/Patches/2009-03-WhoWatchesTheVirindi/9 WeenieDefaults/Creature/Tumerok/40495 Hea Apostate Shock Trooper.sql [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+INSERT INTO `weenie_properties_float` (`object_Id`, `type`, `value`)
+VALUES (40495,   1,       5) /* HeartbeatInterval */
+     , (40495,   2,       0) /* HeartbeatTimestamp */
+     , (40495,   3,      24) /* HealthRate */
+     , (40495,   4,      23) /* StaminaRate */
+     , (40495,   5,       8) /* ManaRate */
+     , (40495,  12,       0) /* Shade */
+     , (40495,  13,       1) /* ArmorModVsSlash */
+     , (40495,  14,       1) /* ArmorModVsPierce */
+     , (40495,  15,       1) /* ArmorModVsBludgeon */
```
Comment (DridsGhost) [inline Database/Patches/2009-03-WhoWatchesTheVirindi/9 WeenieDefaults/Creature/Tumerok/40495 Hea Apostate Shock Trooper.sql:41]: Suggest updating resistances

File: Database/Patches/2009-05-SecretsOfTheApostates/8 QuestDefDB/ApostateNexusFlag - Copy.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+DELETE FROM `quest` WHERE `name` = 'NexusCrawlStarted0509';
```
Comment (DridsGhost) [inline Database/Patches/2009-05-SecretsOfTheApostates/8 QuestDefDB/ApostateNexusFlag - Copy.sql:1]: whats up with the file name - Copy.sql

File: Database/Patches/2009-05-SecretsOfTheApostates/9 WeenieDefaults/Clothing/Armor/40801 Nexus Crawler's Mask.sql [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+     , (40801,   5,        200) /* EncumbranceVal */
+     , (40801,   9,          1) /* ValidLocations - HeadWear */
+     , (40801,  16,          1) /* ItemUseable - No */
+     , (40801,  18,          1) /* UiEffects - Magical */
+     , (40801,  19,          0) /* Value */
+     , (40801,  28,        400) /* ArmorLevel */
+     , (40801,  33,          1) /* Bonded - Bonded */
+     , (40801,  93,       1044) /* PhysicsState - Ethereal, IgnoreCollisions, Gravity */
+     , (40801, 106,        325) /* ItemSpellcraft */
+     , (40801, 107,       1698) /* ItemCurMana */
```
Comment (DridsGhost) [inline Database/Patches/2009-05-SecretsOfTheApostates/9 WeenieDefaults/Clothing/Armor/40801 Nexus Crawler's Mask.sql:18]: update Current Mana

File: Database/Patches/2009-05-SecretsOfTheApostates/9 WeenieDefaults/Creature/HollowMinion/40802 Apostate Nexus Hollow Minion.sql [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+     , (40802,   6,         -1) /* ItemsCapacity */
+     , (40802,   7,         -1) /* ContainersCapacity */
+     , (40802,  16,          1) /* ItemUseable - No */
+     , (40802,  25,        220) /* Level */
+     , (40802,  68,          3) /* TargetingTactic - Random, Focused */
+     , (40802,  72,         19) /* FriendType - Virindi */
+     , (40802,  93,       1032) /* PhysicsState - ReportCollisions, Gravity */
+     , (40802, 133,          2) /* ShowableOnRadar - ShowMovement */
+     , (40802, 140,          1) /* AiOptions - CanOpenDoors */
+     , (40802, 146,     250000) /* XpOverride */;
```
Comment (DridsGhost) [inline Database/Patches/2009-05-SecretsOfTheApostates/9 WeenieDefaults/Creature/HollowMinion/40802 Apostate Nexus Hollow Minion.sql:19]: Should be?

1400000) /* XpOverride

File: Database/Patches/2009-05-SecretsOfTheApostates/9 WeenieDefaults/Creature/HollowMinion/40802 Apostate Nexus Hollow Minion.sql [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+VALUES (40802,   1,       5) /* HeartbeatInterval */
+     , (40802,   2,       0) /* HeartbeatTimestamp */
+     , (40802,   3,       1) /* HealthRate */
+     , (40802,   4,     8.5) /* StaminaRate */
+     , (40802,   5,       1) /* ManaRate */
+     , (40802,  12,       0) /* Shade */
+     , (40802,  13,    0.76) /* ArmorModVsSlash */
+     , (40802,  14,    0.76) /* ArmorModVsPierce */
+     , (40802,  15,    0.68) /* ArmorModVsBludgeon */
+     , (40802,  16,    0.84) /* ArmorModVsCold */
```
Comment (DridsGhost) [inline Database/Patches/2009-05-SecretsOfTheApostates/9 WeenieDefaults/Creature/HollowMinion/40802 Apostate Nexus Hollow Minion.sql:41]: Arent these mobs vulnerable to cold?

**Review by DridsGhost — COMMENTED** [review]
Great work here. Just a few comments.

File: Database/Patches/2009-03-WhoWatchesTheVirindi/9 WeenieDefaults/Creature/Tumerok/40495 Hea Apostate Shock Trooper.sql (hunk 1 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-03-WhoWatchesTheVirindi/9 WeenieDefaults/Creature/Tumerok/40495 Hea Apostate Shock Trooper.sql]: Fixed.

File: Database/Patches/2009-05-SecretsOfTheApostates/8 QuestDefDB/ApostateNexusFlag - Copy.sql (hunk 2 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-05-SecretsOfTheApostates/8 QuestDefDB/ApostateNexusFlag - Copy.sql]: Fixed.

File: Database/Patches/2009-05-SecretsOfTheApostates/9 WeenieDefaults/Clothing/Armor/40801 Nexus Crawler's Mask.sql (hunk 3 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-05-SecretsOfTheApostates/9 WeenieDefaults/Clothing/Armor/40801 Nexus Crawler's Mask.sql]: Fixed.

File: Database/Patches/2009-05-SecretsOfTheApostates/9 WeenieDefaults/Creature/HollowMinion/40802 Apostate Nexus Hollow Minion.sql (hunk 4 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-05-SecretsOfTheApostates/9 WeenieDefaults/Creature/HollowMinion/40802 Apostate Nexus Hollow Minion.sql]: Fixed.

File: Database/Patches/2009-05-SecretsOfTheApostates/9 WeenieDefaults/Creature/HollowMinion/40802 Apostate Nexus Hollow Minion.sql (hunk 5 above)
Comment (sLackeR-hub) [reply Database/Patches/2009-05-SecretsOfTheApostates/9 WeenieDefaults/Creature/HollowMinion/40802 Apostate Nexus Hollow Minion.sql]: Fixed.

**Review by DridsGhost — APPROVED** [review]
Test and approved.

**Pull request merged**